### PR TITLE
Separated CommitmentContract

### PR DESCRIPTION
**Pull request by Akira-Taniguchi**
# description

Separated CommitmentContract.

# why

created a simple contract that retains internal properties, making it difficult for glitches and additional features to occur.

### CODE REVIEW

**Comment by Akira-Taniguchi** [comment]
@syuhei176 How about this one?

**Comment by syuhei176** [comment]
@Akira-Taniguchi @tkmct IMO, it's good to understanding to use nouns for contract name.
for example "Verifier". What do you think?

**Comment by Akira-Taniguchi** [comment]
I think it's a good idea.

Thank you.


We'll fix it at 9:30.

@syuhei176

File: scripts/deploy.ts [hunk 1]
```diff
@@ -463,9 +471,10 @@ const deployContracts = async (
     constantVariableTable: {
       secp256k1: encodeString('secp256k1'),
       txAddress: txAddress,
-      commitmentContract: encodeAddress(commitmentContract.address)
```
Comment (syuhei176) [inline scripts/deploy.ts:466]: ah, it should be `verifyCommitmentData.address`.

Comment (Akira-Taniguchi) [reply scripts/deploy.ts:428]: It has been finished, please check. @syuhei176

File: scripts/deploy.ts [hunk 2]
```diff
@@ -425,9 +433,9 @@ const deployContracts = async (
...
+      verify: encodeAddress(verify.address)
```
Comment (syuhei176) [inline scripts/deploy.ts:436]: key is "commitmentContract"


```
commitmentContract: encodeAddress(verify.address)
```

File: contracts/Verify.sol [hunk 3]
```diff
@@ -2,49 +2,24 @@ pragma solidity ^0.5.0;
...
  * @notice This is mock commitment chain contract. Spec is http://spec.plasma.group/en/latest/src/02-contracts/commitment-contract.html
  */
-contract CommitmentContract is Storage {
-    // Single operator address
-    address public operatorAddress;
-    // Current block number of commitment chain
-    uint256 public currentBlock = 0;
-    // History of Merkle Root
-    mapping(uint256 => bytes32) public blocks;
+contract Verify is Storage {
```
Comment (syuhei176) [inline contracts/Verify.sol:12]: I recommend CommitmentVerifier

**Comment by Akira-Taniguchi** [comment]
Sorry for repeating myself, it's done.
@syuhei176

**Pull request merged**